packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -686,7 +686,8 @@ class SliverGeometry extends Diagnosticable {
   ///  * [RenderViewport.cacheExtent] for a description of a viewport's cache area.
   final double cacheExtent;
 
-  static const double _epsilon = 1e-10;
+  /// The epsilon of tolerable double precision error.
+  static const double precisionErrorTolerance = 1e-10;
```

Comment:
we really shouldn't expose this in our public API. This kind of thing is a hack at best.

Reply:
I am happy to revert this is this is not preferable. This issue has been happened in a lot different place. There is another bug in performlayout I am trying to fix .  https://github.com/flutter/flutter/pull/30979/files
I think it is better to have a central place to define this constant. That is why I exposed this to public api.

Reply:
I guess it's better to expose it and use it consistently than to have it in many places...